### PR TITLE
Create JobDetails component

### PR DESCRIPTION
**Pull request by bernardodiasc**
**Release Type:** *Non-breaking feature*

Fixes https://x-team-internal.atlassian.net/browse/XP-3372

## Description

Create JobDetails component.

> **Notice 1:** I spent many hours trying to make the `blockquote` to render as expected. I could not resolve the problem. The documentation is clear https://github.com/probablyup/markdown-to-jsx/blob/master/fixture.md#blockquotes, there's nothing wrong in the code as far as I can see...
> **Notice 2:** I also tried to resolve the problem with HTML Entities being displayed inside code blocks (e.g. `>` is rendered as `&gt;`).
> 
> As the issues described are kinda not under the expectations (because the expectations is vague and Figma file have a limited amount of textual HTML elements), I believe we could/should raise follow ups to address those bugs if they are in fact issues for the users.

## Checklist

- [x] set yourself as an assignee
- [x] set appropriate labels for a PR (`In Review` or `In Progress` depending on its status)
- [x] move respective JIRA issue to the `IN REVIEW` column
- [x] make sure your code lints (`npm run lint`)
- [x] Flow typechecks passed (`npm run flow`)
- [x] Snapshots tests passed (`npm run jest`)
- [x] component's documentation (`.stories.js` file) is changed or added accordingly to reflect any new or updated use cases or variants usage
- [x] if any snapshots have been changed, verify that component still works and looks as expected and update the changed snapshot
- [x] **manually tested the app** by running it in several different browsers (Firefox, Chrome, Opera, Safari, MS IE/Edge, etc.) and checked nothing is broken and operates as expected!

## Steps to Test or Reproduce

1. Run `npm start`
1. Go to http://localhost:9001/?path=/story/ui-components-xp-registration-jobdetails--default
1. Feel free to edit Knobs to test different markdown content

## Impacted Areas in Application

- `JobDetails` was created

## Screenshots

Screen Shot 2019-10-17 at 13 18 59

### CODE REVIEW

File: src/components/ui/JobDetails.stories.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+>
+> Here's some example code:
+>
+>     return shell_exec("echo $input | $markdown_script");
+>
+> ${faker.lorem.paragraph()}
+> ${faker.lorem.paragraph()}
+`
+
+storiesOf('UI Components/XP-Registration/JobDetails', module)
```
Comment (rayepps) [inline src/components/ui/JobDetails.stories.js]: May want to update these to match our latest storybook changes. I don't think we're still using the `XP-Registration` folder.

Comment (bernardodiasc) [reply src/components/ui/JobDetails.stories.js]: Good catch, will do the update once https://github.com/x-team/xp-ui/pull/460 got merged 👍

**Comment by nicksp** [comment]
@rayepps Could you please resolve merge conflicts here? Rebuilding the project should help out.

**Comment by nicksp** [comment]
@bernardodiasc Could you please resolve conflicts? Rebuilding a project should help out. LMK if any help needed.

**Pull request merged**